data_models/aws_vpcflow_data_model.yml

```diff
@@ -0,0 +1,18 @@
+AnalysisType: datamodel
+LogTypes: 
+  - AWS.VPCFlow
+DataModelID: AWS.VPCFlow.DataModel
+Enabled: true
+Mappings:
+  - Name: actor_user
+    Path: $.userIdentity.userName
```

Comment:
I don't think VPC Flow Logs have a `userIdentity` field, is this part of some expanded field set or something I'm not familiar with maybe?

Reply:
Nope, I just mistakenly added it here - perhaps from copying another aws data model to this one. Removed! :)